.buildkite/pipeline.yml

```diff
@@ -145,18 +145,31 @@ steps:
     env:
       TMPDIR: "/cache"
 
+  - block: "macOS steps"
```

Comment:
Is the reason to call it `macOS steps` as opposed to `Build package (macOS)` the nix check being included as well?

Reply:
Not quite. The reason is that the `Run unit tests (macOS)` step depends on the `Check nix (macOS)` step.

This implies that `macOS steps` must be clicked before clicking `macOS test` has an effect.

I don't entirely. But I don't quite have a good idea on how to improve it. 🤔

Reply:
> I don't quite like this — but I don't quite have a good idea on how to improve it either.

We could just move them all under one `macOS checks` prompt, like we did in #3688, avoiding the oddness you mentioned at the cost of reduced granularity we likely mostly don't need.

Reply:
Agree, I will do that. 👍